test-utils/build.gradle

```diff
@@ -29,7 +29,7 @@ dependencies {
     api group: 'org.hamcrest', name: 'hamcrest-library', version: '2.2'
     api group: 'org.neo4j.community', name: 'it-test-support', version: neo4jVersionEffective // , classifier: "tests"
     api group: 'org.neo4j', name: 'log-test-utils', version: neo4jVersionEffective // , classifier: "tests"
-    api group: 'org.neo4j.driver', name: 'neo4j-java-driver', version: '4.0.0'
+    api group: 'org.neo4j.driver', name: 'neo4j-java-driver', version: '4.4.12'
```

Comment:
Shouldn't this be a 5.x driver? https://mvnrepository.com/artifact/org.neo4j.driver/neo4j-java-driver/5.13.0

Reply:
I believe the driver API is breaking between 4 and 5, so lets make this bump to LTS and take it to 5 another day.